File: Models/SocialCare.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace MtpApp.Models
{
    public class SocialCare
    {
        public int Id { get; set; }
        [Required]
        public int ClientId { get; set; }
        public bool SocialSupportNeeded { get; set; }
        public bool MeetingFuture { get; set; }
        public bool Problem { get; set; }
    }
}

File: spring-backend/src/main/java/com/mtp/api/models/SocialCare.java
package com.mtp.api.models;

import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Entity
@Table(name = "SocialCares")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class SocialCare {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @NotNull
    private Integer clientId;

    private boolean socialSupportNeeded;
    private boolean meetingFuture;
    private boolean problem;
}
